websites/T/Telegram/presence.ts

```diff
@@ -5,6 +5,8 @@ var presence = new Presence({
 presence.on("UpdateData", async () => {
   const path = document.location.href;
 
+  const showName = await presence.getSetting("name");
+
   const presenceData: PresenceData = {
```

Comment:
merge consts thanks

Reply:
Explain please

Reply:
bruh
```javascript
const f = "fgf",
g = "sgsg",
h = "afaf";
```

Reply:
const path = document.location.href,
showName = await presence.getSetting("name");
?

Reply:
you have three consts to merge